e2e/tests/upgrades/upgrade_test.go

```diff
@@ -74,18 +74,13 @@ func (s *UpgradeTestSuite) UpgradeChain(ctx context.Context, chain *cosmos.Cosmo
 		Info:   fmt.Sprintf("upgrade version test from %s to %s", currentVersion, upgradeVersion),
 	}
 
-	if testvalues.GovV1MessagesFeatureReleases.IsSupported(chain.Config().Images[0].Version) {
```

Comment:
Unsure if this is really required for tests anymore. If something from v0.47 (ibc-go/v7) fails then this would be the place to look.

For now I think moving forward with gov/v1 is the way to go